<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartEditDialog</class>
 <widget class="QDialog" name="PartEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Запчасть</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editPartName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Артикул:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editPartNumber"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Категория:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboCategory">
       <item>
        <property name="text">
         <string>Процессор</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Оперативная память</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Жесткий диск</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SSD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Материнская плата</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Видеокарта</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Блок питания</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Корпус</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Охлаждение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Кабель</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Аксессуар</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Другое</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Количество:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinQuantity">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Цена (руб):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="spinPrice">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Мин. запас:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinMinQuantity">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>